openslides/users/models.py

```diff
@@ -150,3 +160,83 @@ def reset_password(self, password=None):
         if password is None:
             password = self.default_password
         self.set_password(password)
+
+
+def gen_username(first_name, last_name):
```

Comment:
this is not a model. Please make this either a method of the User model or the user manager, or find another place. Django uses often the file 'utls' for this.